frontend/ui/data/onetime_migration.lua

```diff
@@ -10,7 +10,7 @@ local util = require("util")
 local _ = require("gettext")
 
 -- Date at which the last migration snippet was added
-local CURRENT_MIGRATION_DATE = 20240928
+local CURRENT_MIGRATION_DATE = 20241117
```

Comment:
(Update the date to the day this gets a chance to be merged :))

Reply:
mmm what does that mean? today? I thought the specific date was irrelevant so long as it was a number larger than the previously used one.

Reply:
it indeed just needs to be larger than the previous one - but best to have it precise so we know when things happened and are not misleaded.
Today is fine if the PR is ready (dunno about @hius07 last comment), even if merged tomorrow.